src/main/java/Owner.java

```diff
@@ -0,0 +1,75 @@
+public class Owner {
+    int ID;
+    String name;
+    String surname;
+    String email;
+    int phone;
```

Comment:
Phone number should be represented by String, you should think about cases in other countries: (123) 123 23 23 or +48 123456789
https://stackoverflow.com/questions/42255754/phone-number-should-be-a-string-or-some-numeric-type-that-have-capacity-to-save